src/tunnel.c

```diff
@@ -322,9 +326,22 @@ static int tcp_connect(struct tunnel *tunnel)
 		} else {
 			server.sin_port = htons(tunnel->config->gateway_port);
 		}
+		// get rid of a trailing slash
+		if (proxy_host[strlen(proxy_host) - 1] == '/')
+			proxy_host[strlen(proxy_host) - 1] = '\0';
 		log_debug("proxy_host: %s\n", proxy_host);
 		log_debug("proxy_port: %s\n", proxy_port);
 		server.sin_addr.s_addr = inet_addr(proxy_host);
+		// if host is given as fqhn we have to do a dns lookup
+		if (server.sin_addr.s_addr == INADDR_NONE) {
+			host = gethostbyname(proxy_host);
```

Comment:
While _gethostname()_ is perfect right now, please note for future reference that pull request #77 suggests switching to _getaddrinfo()_. It makes sense as [_gethostname()_ is obsolete and not part of POSIX anymore](https://github.com/adrienverge/openfortivpn/pull/77#issuecomment-335367313).